Packs/AbuseDB/Integrations/AbuseDB/AbuseDB.py

```diff
@@ -123,14 +123,16 @@ def analysis_to_entry(info, reliability, threshold=THRESHOLD, verbose=VERBOSE):
     for analysis in info:
         ip_ec = {
             "Address": analysis.get("ipAddress"),
-            "Geo": {"Country": analysis.get("countryName") or analysis.get("countryCode")}
+            "Geo": {"Country": analysis.get("countryName") or analysis.get("countryCode")},
+            "ISP": analysis.get("isp")
```

Comment:
we do not have this indicator field IIRC. so if you wish to add it, I think it hsould go with an indicator field and probably checked with the security team.